packages/core/types-internal/src/index.js

```diff
@@ -288,7 +327,7 @@ export type DetailedReportOptions = {|
 
 declare type GlobPattern = string;
 
-export type InitialParcelOptions = {|
+export type InitialParcelOptionsInternal<WorkerFarm> = {|
```

Comment:
This is a breaking change for this package. Is there a way we can retain the old type?